git_machete/exceptions.py

```diff
@@ -21,7 +21,7 @@ class UnprocessableEntityHTTPError(Exception):
     or creating a pull request for a branch that already has a PR.
     """
     def __init__(self, msg: str) -> None:
-        self.msg: str = msg
+        self.msg: str = msg + "\nPlease open an issue regaring this topic under link: https://github.com/VirtusLab/git-machete/issues/new"
```

Comment:
`regarding`

Reply:
Hmmm but wait... isn't 422 a perfectly predicted case (when `assigning someone from outside organization as a reviewer or creating a pull request for a branch that already has a PR`)? should a user create an issue in such case?

Reply:
Yes, then should be ticket opened. Error 422 tells that operation is not allowed by GitHub for some reason. `assigning someone from outside organization as a reviewer or creating a pull request for a branch that already has a PR` are just known possible reasons to get 422. I expect to be more of them, especially in functions which use `PATCH` method. AFAICS all those errors could be avoided by simple checks, like it is right now in case of `creating a pull request for a branch that already has a PR`. `assigning someone from outside organization` could also be avoided by checking in GitHub API who is in organization (I'll open an issue for this as soon as this PR is merged). In conclusion every 422 error is a great base to open a ticket.

Reply:
In case we cannot do anything with 422, it should be caught by higher lever function.

Reply:
Okay! thanks for the above clarification, I've got lost a bit.